tools/compiler/temp/example.go

```diff
@@ -0,0 +1,34 @@
+package temp
+
+import (
+	"fmt"
+	"github.com/goplus/gop/ast"
+	"github.com/goplus/gop/parser"
+	"github.com/goplus/gop/token"
+)
+
+func main() {
+	// 源代码
```

Comment:
don't use Chinese.

Reply:
sorry, i'll delete those chinse comment in this example code latter.